src/main/webapp/style.css

```diff
@@ -129,3 +129,7 @@ footer {
   text-align: center;
   width: 100%;
 }
+
+#colors {
+  text-align: center;
+}
```

Comment:
Taking a look through this css, we have text-align: center on almost everything. Would it be possible to just apply it to one larger thing (e.g. the root) instead? (I'm not sure if this will work or not, so might have to experiment.)